src/schemas/addMealSchema.js

```diff
@@ -0,0 +1,10 @@
+import * as yup from 'yup';
+
+import { MIN_VALUE_ERROR, MUST_BE_NUMBER, REQUIRED_ERROR } from './const';
+
+export const addMealSchema = yup.object().shape({
+  title: yup.string().required(REQUIRED_ERROR),
+  description: yup.string().required(REQUIRED_ERROR),
+  weight: yup.number(MUST_BE_NUMBER).positive(MIN_VALUE_ERROR).required(REQUIRED_ERROR),
+  price: yup.number(MUST_BE_NUMBER).positive(MIN_VALUE_ERROR).required(REQUIRED_ERROR),
```

Comment:
The 2 above could use `yup.number().typeError(MUST_BE_NUMBER)` instead as currently if you leave the number inputs empty, the error message is `must be a `number` type, but the final value was: `NaN` (cast from the value `""`).`.